components/p3a/buildflags.gni

```diff
@@ -1,3 +1,4 @@
 declare_args() {
   brave_p3a_enabled = true
+  brave_p3a_android_browsertest = false
```

Comment:
whatever we end up doing for the P3A thing, this is definitely not going to work because it is going to cause a rebuild which will also screw up the output apk so this is not a viable solution